Mark toolkits that require custom auth and Callout auth providers without a default app (#113)
ArcadeAI/docs: pages/home/auth-providers/atlassian.mdx

# Atlassian auth provider

> <Note>
>   At this time, Arcade does not offer a default Atlassian auth provider. To use this auth provider, you must create your own custom Auth Provider by providing your own Atlassian app credentials.
> </Note>
> 

Reviewer: ```suggestion
<Note>
  At this time, Arcade does not offer a default Atlassian auth provider. To use Atlassian auth, you must create a custom Auth Provider with your own Atlassian app credentials.
</Note>

```
